src/app/dev.js

```diff
@@ -14,90 +15,89 @@ import { getModuleContext, getChanged } from './util/webpack';
  * Register all templates beforehand with a require context, and save the partial by filename
  */
 const [partialModules, partialsContext] = getModuleContext(
-	require.context('./component/blocks/', true, /\.hbs$/),
-	(context, key, module) => {
-		Handlebars.registerPartial(/\/([^/]+)\.hbs/gi.exec(key)[1], module)
-	}
+  require.context('./component/blocks/', true, /\.hbs$/),
+  (context, key, module) => {
+    Handlebars.registerPartial(/\/([^/]+)\.hbs/gi.exec(key)[1], module);
+  },
```

Comment:
`(context, key, module) => Handlebars.registerPartial(/\/([^/]+)\.hbs/gi.exec(key)[1], module),`

Reply:
Technically the function doesn't return anything, so using a function body is more correct IMO.

Reply:
Agreed.